izpack-installer/src/main/java/com/izforge/izpack/installer/gui/InstallerFrame.java

```diff
@@ -304,6 +305,9 @@ public void buildGUI()
 
         Messages messages = locales.getMessages();
         navigator.updateButtonText(messages);
+        
+        // set the frame title
+        setTitle(getTitle(installdata));
```

Comment:
It is not necessary to pass the installdata as argument to getTitle(...), is it?
The field installdata initialized in the constructor should be accessed directly in getTitle().
Could you please do this small change before merging?